tools/files/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - stable-*
+    tags:
+      - "*"
+
+jobs:
+  linters:
+    uses: ./.github/workflows/linters.yml # use the callable linters job to run tests
+  ansible-lint:
```

Comment:
I think we should either remove `ansible-lint` from these jobs or remove it from the `linters` job, otherwise we are running it twice. Since the ansible-content-actions ansible_lint workflow doesn't version ansible-lint or allow version specification, I would vote to remove that workflow and use the one declared in our `linters` workflow.

Reply:
Are we going to remove `ansible-lint` here?

Reply:
No. I thought I removed it. I shall fix it.